docs/vision_challenges.rst

```diff
@@ -0,0 +1,82 @@
+.. _vision_challenges:
+
+Vision & Challenges
+====================
+
+As mentioned in the :ref:`introduction`, an overarching goal is to make research
+software [FAIR](https://www.go-fair.org/fair-principles/). In this section, we
+want to discuss possible levels of FAIRness associated with the code and data that
```

Comment:
By the way, I used "level" as a placeholder for now. It is probably not the word we want to use, I just couldn't yet come up with a better one.